README.md

```diff
@@ -4,13 +4,57 @@ A new Flutter project.
 
 ## Getting Started
 
-This project is a starting point for a Flutter application.
+This configuration strategy relies on the `--dart-define` command-line argument to define compile-time environment 
+variables. The default values could be used for local development purposes, and CI/CD tools can override the values 
+depending on the environment being deployed to.
 
-A few resources to get you started if this is your first Flutter project:
+Simply run:
 
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+```sh
+flutter run  # to rely on the default 'multiplier' value, or
```

Comment:
Will Flutter run work in our ci/cd pipelines? There are no devices so I think it may fail. Can we also do `--dart-define` for the `flutter build` calls?

Reply:
These instructions are aimed at other devs who are 'Getting Started' with this repo. They can clone it, and they should only need to do `flutter run` (assuming they have the Flutter SDK installed), or `flutter run --dart-define multiplier=2` to see the results of using `--dart-define`.

`flutter test` will be easy to run on build agents, and will be enough for our unit- / component-level testing. If we want instrumented tests running on the build agents, then we'll need a solution that's capable of running an emulator.

Reply:
You know I can't find any winning documentation that states authoritatively that `flutter build apk --dart-define customArg=42` is legal, but when running the commands locally:

```sh
flutter build apk --dart-define multiplier=2  # builds successfully
flutter build apk --dart-dene multiplier=2  # fails because option "dart-dene" cannot be found
```

Then, when I install the built APK file onto an emulator and run the app, the value of `multiplier` is indeed `2`. So looks like we're good here too (phew, 'cause otherwise that would have made `--dart-define` completely pointless for us).

Reply:
Awesome that it also seems to work for `build apk`. That's looking good.